tests/scripts/beneficiary/delete_or_suspend_account_from_csv_test.py

```diff
@@ -0,0 +1,36 @@
+import pytest
+
+from pcapi.core.users.factories import FavoriteFactory
+from pcapi.core.users.factories import UserFactory
+from pcapi.models import Deposit
+from pcapi.models import User
+from pcapi.repository import repository
+from pcapi.scripts.beneficiary.delete_or_suspend_account_from_csv import run
+
+
+@pytest.mark.usefixtures("db_session")
+def test_delete_user():
+    UserFactory(email="romain.chaffal@beta.gouv.fr", isAdmin=True)
```

Comment:
```suggestion
    admin = UserFactory(isAdmin=True)
```

(Accessoirement, ça évite d'avoir des vrais noms de domaine et des vrais noms de personne dans le code...)